README.md

```diff
@@ -1,37 +1,64 @@
+    </picture>
+  </a><br /><br />
+</p>
+ 
+# Meshery Adapter for Nighthawk
+<div align="center">
 
-## Join the service mesh community!
+[![Docker Pulls](https://img.shields.io/docker/pulls/layer5/meshery-nighthawk.svg)](https://hub.docker.com/r/layer5/meshery-nighthawk)
+[![Go Report Card](https://goreportcard.com/badge/github.com/meshery/meshery-nighthawk)](https://goreportcard.com/report/github.com/meshery/meshery-nighthawk)
+[![Build Status](https://img.shields.io/github/actions/workflow/status/meshery/meshery-nighthawk/multi-platform.yml?branch=master)](https://github.com/meshery/meshery-nighthawk/actions)
+[![GitHub](https://img.shields.io/github/license/meshery/meshery-nighthawk.svg)](LICENSE)
+[![GitHub issues by-label](https://img.shields.io/github/issues/meshery/meshery-nighthawk/help%20wanted.svg)](https://github.com/meshery/meshery-nighthawk/issues?q=is%3Aopen+is%3Aissue+label%3A"help+wanted")
+[![Website](https://img.shields.io/website/https/layer5.io/meshery.svg)](https://layer5.io/meshery/)
+[![Twitter Follow](https://img.shields.io/twitter/follow/layer5.svg?label=Follow&style=social)](https://twitter.com/intent/follow?screen_name=mesheryio)
+[![Discuss Users](https://img.shields.io/discourse/users?server=https%3A%2F%2Fdiscuss.layer5.io)](https://discuss.layer5.io)
+[![Slack](https://img.shields.io/badge/Slack-@layer5.svg?logo=slack)](http://slack.layer5.io)
+[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/3564/badge)](https://bestpractices.coreinfrastructure.org/projects/3564)
 
-<a name="contributing"></a><a name="community"></a>
-Our projects are community-built and welcome collaboration. 👍 Be sure to see the <a href="https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit">Layer5 Community Welcome Guide</a> for a tour of resources available to you and jump into our <a href="http://slack.layer5.io">Slack</a>!
+</div>
 
 <p style="clear:both;">
-<a href ="https://layer5.io/community/meshmates"><img alt="MeshMates" src=".github/readme/images/Layer5-MeshMentors-1.png" style="margin-right:10px; margin-bottom:15px;" width="28%" align="left"/></a>
-<h3>Find your MeshMate</h3>
+<h2><a href="https://layer5.io/meshery">Meshery</a></h2>
+<a href="https://meshery.io"><img src="img/readme/meshery-logo-light-text-tag.svg"
+style="margin:10px;" width="125px" 
+alt="Meshery - the Collaborative Cloud Manager" align="left" /></a>
+As a self-service engineering platform, <a href="https://meshery.io">Meshery</a> enables collaborative design and operation of cloud native infrastructure. Through it's extension points, Meshery offers the ability to optionally plugin adapters in order to more deeply integrate with specific systems like <a href="https://getnighthawk.io">Nighthawk</a>.
```

Comment:
https://getnighthawk.io can't be accessed, it should be https://getnighthawk.dev.

Reply:
```suggestion
As a self-service engineering platform, <a href="https://meshery.io">Meshery</a> enables collaborative design and operation of cloud native infrastructure. Through it's extension points, Meshery offers the ability to optionally plugin adapters in order to more deeply integrate with specific systems like <a href="https://getnighthawk.dev">Nighthawk</a>.
```

Reply:
Oh! Thank you, @gyohuangxin 👍